src/user_data/value_information.rs

```diff
@@ -1041,6 +1041,8 @@ pub enum ValueLabel {
     DisplayOutputScalingFactor,
     ManufacturerSpecific,
     Volume,
+    FlowTemperature,
```

Comment:
@dragonnn can you help me here. Somehow I had to add this type. It was not in the code but it was in your test. And because the test ran with "cfg std" it did not fail in the pipeline.